components/bookshelf/Book.tsx

```diff
@@ -42,6 +46,15 @@ const Book = ({ book, reading = false }: BookProps) => {
     removeFromList(book.user_book_id)
   }
 
+  const unmarkBook = () => {
+    console.log("book unmarked")
```

Comment:
Can you put an action instead of console log here?

Reply:
yep. will do on a seperate PR